libcontainer/intelrdt/intelrdt.go

```diff
@@ -458,6 +458,7 @@ func (m *Manager) Apply(pid int) (err error) {
 	}
 
 	path := m.GetPath()
+	fmt.Println("intelrdt: applying intelrdt path:", path, "pid:", pid)
```

Comment:
no fmt, if you want log you can use the logging lib

Reply:
Doh, some leftovers from development. Removed